[CLI] Add `hf discussions edit` (#4415)
huggingface/huggingface_hub: docs/source/en/guides/cli.md

>>> echo "LGTM" | hf discussions comment username/my-model 5 --body-file -
```

### Edit a comment

> Edit an existing comment in place by passing the discussion number and the comment ID. Comment IDs are available in the `events` array of `hf discussions info ... --format json`:

Author: ```suggestion
Edit an existing comment in place by passing the discussion number and the comment ID. Comment IDs can be retrieved with `hf discussions info`:
```
